components/SideBar/SideBarSavedPosts.tsx

```
@@ -6,45 +6,43 @@ import SideBarSavedArticlePreview from "./SideBarSavedArticlePreview";
 
   return (
     <div className="w-full">
       <h3 className="mb-4 mt-8 text-2xl font-semibold leading-6 tracking-wide">
         Recent bookmarks
       </h3>
       <div className="w-full">
-        {bookmarkStatus === "loading" &&
+        {status === "loading" &&
           Children.toArray(
             Array.from({ length: howManySavedToShow }, () => {
               return <LoadingSkeleton />;
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)